common/verify_report_utils/verify_report/verify_attestation_report.py

```diff
@@ -20,7 +20,7 @@
 logger = logging.getLogger(__name__)
 
 
-def verify_attestation_report(enclave_info):
+def verify_ias_attestation_report(enclave_info):
```

Comment:
Can we rename this file to `verify_ias_attestation_report.py` if this file is specific to IAS ?

Reply:
Moved to different directory.